app/http.php

```diff
@@ -501,6 +501,41 @@ function cleanUp(Orchestration $orchestration, Table $activeRuntimes, array $net
         $response->end();
     });
 
+Http::post('/v1/runtimes/:runtimeId/commands')
+    ->desc('Execute a command inside an existing runtime')
+    ->param('runtimeId', '', new Text(64), 'Unique runtime ID.')
+    ->param('command', '', new Text(1024), 'Command to execute.')
+    ->param('timeout', 600, new Integer(), 'Commands execution time in seconds.', true)
```

Comment:
Let's have a test when we do "sleep 5 && echo "OK"", and ensure response output doesnt have OK, when timeout is set to 1.
Also ensure there is some 4XX or 5XX response, not 2XX